src/esp/assets/GenericMeshData.cpp

```diff
@@ -36,26 +36,26 @@ void GltfMeshData::uploadBuffersToGPU(bool forceReload) {
   buffersOnGPU_ = true;
 }
 
-Magnum::GL::Mesh* GltfMeshData::getMagnumGLMesh() {
+Magnum::GL::Mesh* GenericMeshData::getMagnumGLMesh() {
   if (renderingBuffer_ == nullptr) {
     return nullptr;
   }
 
   return &(renderingBuffer_->mesh);
 }
 
-void GltfMeshData::setMeshData(Magnum::Trade::AbstractImporter& importer,
-                               int meshID) {
-  ASSERT(0 <= meshID && meshID < importer.meshCount());
+void GenericMeshData::setMeshData(Magnum::Trade::AbstractImporter& importer,
+                                  int meshID) {
+  /* Guarantee mesh instance success */
+  CORRADE_INTERNAL_ASSERT_OUTPUT(meshData_ = importer.mesh(meshID));
```

Comment:
```suggestion
  CORRADE_INTERNAL_ASSERT(0 <= meshID && meshID < importer.meshCount());
  
  /* Guarantee mesh instance success */
  CORRADE_INTERNAL_ASSERT_OUTPUT(meshData_ = importer.mesh(meshID));
```

Somehow the original assert got lost? Tho Magnum will provide a helpful assertion in this case also, so it's probably redundant to check for meshID bounds here as well.

Reply:
Oh, I specifically removed it thinking that it addressed all 3 of the original lines of code.  Should i replace the range check?